protocols/v2/binary-sv2/no-serde-sv2/codec/src/codec/encodable.rs

```diff
@@ -1,3 +1,64 @@
+// This module provides an encoding framework for serializing various data types into bytes.
```

Comment:
Same story for this, because of this module encapsulation the top level doc on this `pub mod encodable {
    pub use crate::codec::encodable::{Encodable, EncodableField, EncodablePrimitive};
}` is not rendered, and we have add the docs in lib.rs